### DISCUSSION TITLE
Release v0.1.19

### DISCUSSION THREAD

**Discussion by ehendrix23** [ehendrix23/tesla_dashcam#170] [Announcements]
- Fixed:
    - Tracebacks when there is no events.json file issue #168 
    - Exit going through preferences within RunPreferences if an execution of tesla_dashcam failed

<hr /><em>This discussion was created from the release <a href='https://github.com/ehendrix23/tesla_dashcam/releases/tag/v0.1.19'>Release v0.1.19</a>.</em>

**Comment by Bonscha** [comment]
I'm working on a simple Windows GUI for myself. You can package that up with your tool if you want to. Just let me know. :)

**Reply by Bonscha** [reply]
![image](https://user-images.githubusercontent.com/92258523/136707746-79bef06c-a0a6-4a5c-b286-8563a7b1f320.png)

short WIP first draft
video preview is static images only
